lib/rules/require-camelcase-or-uppercase-identifiers.js

```diff
@@ -129,6 +129,17 @@ module.exports.prototype = {
                     return;
                 }
 
+                if (prevToken && prevToken.value === ':') {
```

Comment:
It get the job done! Although i wonder if there is better way. Would you mind commenting this out?